bin/desi_emlinefit_afterburner

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python
+
+import os
+import sys
+from time import time
+import fitsio
+from astropy.time import Time
+import numpy as np
+from desiutil.log import get_logger
+from argparse import ArgumentParser
+from desispec.io.emlinefit import (
+    allowed_emnames,
+    get_targetids,
+    get_rf_em_waves,
+    get_emlines,
+)
+
+def get_parse(log=get_logger()):
+    # AR some defaults
+    default_emnames = "OII,HDELTA,HGAMMA,HBETA,OIII,HALPHA"
+    default_rr_keys = "TARGETID,Z,ZWARN,SPECTYPE,DELTACHI2"
+    default_fm_keys = "TARGET_RA,TARGET_DEC,OBJTYPE"
+    #
+    parser = ArgumentParser(description="Simple emission line fitter; primarily designed for ELGs.")
+    parser.add_argument(
+        "--redrockfn",
+        help="full path to a redrock/zbest file (default=None)",
+        type=str,
+        default=None,
+        required=True,
+    )
+    parser.add_argument(
+        "--coaddfn",
+        help="full path to a coadd file (everest-formatted) (default=None)",
+        type=str,
+        default=None,
+        required=True,
+    )
+    parser.add_argument(
+        "--bitnames",
+        help="comma-separated list of target bits to fit from the CMX_TARGET, SV{1,2,3}_DESI_TARGET, or DESI_TARGET mask; if 'ALL', fits all fibers (default=ELG)",
+        type=str,
+        default="ELG",
+        required=False
+    )
+    parser.add_argument(
+        "--outfn",
```

Comment:
Minor, but for consistency with other desispec programs, let's call the I/O options `--redrock`, `--coadd`, and `--output` (like the desi_qso_*_afterburner scripts)

Reply:
done.